defaults/test_default.yaml

```diff
@@ -136,7 +136,6 @@ scylla_version: ''
 test_upgrade_from_installed_3_1_0: false
 target_upgrade_version: ''
 disable_raft: true
```

Comment:
Is this still true?  Do we disable raft by default?

Reply:
IIUC this flag is not relevant anymore, unless still relevant to only 1 out of 2 raft features. @aleksbykov , @roydahan , can you please confirm?

Reply:
Probably not.
@aleksbykov / @temichus ?

Reply:
This flag is only was used for rolling upgrade tests. Job with disabled raft uses additional yaml with relevant